robot/basestation/static/CSS/app.css

```diff
@@ -0,0 +1,143 @@
+html, body {
+    height: 100%;
+    background-color: #990000;
+}
+
+.container {
+    height: 100%;
+    min-height: 100%;
+}
+
+.fill {
+    width: 100%;
+    height: 100%;
+    min-height: 100%;
+    min-width: 100%;
+}
+
+.map {
+    height: 100vh;
+    min-height: 100vh;
+}
+
+.table_grp, .table_grp2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+.table_grp td, .table_grp2 td {
+    font-family: Arial, sans-serif;
+    font-size: 14px;
+    padding: 0;
+    border-style: none;
+    border-width: 0;
+    overflow: hidden;
+    word-break: normal;
+    border-color: #990000;
+}
+
+.table_grp th, .table_grp2 th {
+    font-family: Arial, sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    padding: 0;
+    border-style: solid;
+    border-width: 1px;
+    overflow: hidden;
+    word-break: normal;
+    border-color: #990000;
+    text-align: center;
+}
+
+.table_grp tr:last-child, .autoCtrLable {
+    border-bottom: 5px inset #05072F;
+    color: #6768A1;
+    font-weight: bold;
+    background-color: #101148;
+}
+
+.autoCtrLable {
+    padding: 12px 5px 12px 5px;
+    margin-top: 15px
+}
+
+.row, .row>* {
+    border: 0.5px solid #05072F;
+}
+
+.control_btn {
+    width: 80px;
+    height: 80px;
+    background-color: #4A0000;
+    border: ridge 2px #434586;
+    color: white;
+    border-radius: 10px;
+}
+
+.control_btn2 {
+    width: 80px;
+    height: 80px;
+    background-color: #4A0000;
+    border: ridge 2px #434586;
+    color: white;
+}
+
+.control_btn:hover, .control_btn2:hover, #logs_btn button:hover {
+    background-color: #05072F;
+    color: white;
+}
+
+.control_btn_left {
+    border-radius: 10px 0 0 10px;
+    border-right-color: #4A0000;
+}
+
+.control_btn_right {
+    border-radius: 0 10px 10px 0;
+    border-left-color: #4A0000;
+}
+
+.control_btn_mid {
+    width: 80px;
+    height: 80px;
+    background-color: #4A0000;
+    border-top: ridge 2px #434586;
+    border-bottom: ridge 2px #434586;
+    border-left: ridge 2px #4A0000;
+    border-right: ridge 2px #4A0000;
+    color: white;
+}
+
+.continer_headers, thead {
```

Comment:
I think you meant "container" instead of "continer" here. It would probably be best to use singular form, as in `container-header`.